static/src/controls/FileExplorerVideoModal.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <div class="modal" tabindex="-1" role="dialog">
+      <div class="modal-dialog modal-video" role="document">
+        <div class="modal-content">
+          <div class="modal-header border-bottom-0">
+            <button type="button"
+                    class="close"
+                    data-dismiss="modal"
+                    aria-label="Close">
+            </button>
+          </div>
+          <div class="modal-body">
+            <center>
```

Comment:
The < center > tag is not supported in HTML5. Use CSS instead.
(source https://www.w3schools.com/tags/tag_center.asp)

Il faut utiliser flex : 
J'utilise tout le temps cette cheat sheet qui est bien illustree : https://css-tricks.com/snippets/css/a-guide-to-flexbox/
La cheat sheet to donnera le css a rajouter.

Ensuite, une fois que ca marche, pour ne pas avoir a ecrire notre css specifique, on peut juste utiliser les classes bootstrap :
https://getbootstrap.com/docs/4.0/utilities/flex/

Reply:
J'ai finalement retiré le tag center, et il se trouve que la modal est déjà centrée. Je ne sais pas pourquoi j'avais eu besoin de faire ça.